src/lib/installation/console/plugins/repositories_command.rb

```diff
@@ -0,0 +1,38 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2021 SUSE LLC, All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# ------------------------------------------------------------------------------
+
+require "pp"
+require "yast"
+
+module Installation
+  module Console
+    # define the "repositories" command in the expert console
+    class Commands
+      def repositories
+        Yast.import "Pkg"
+
+        repos = Yast::Pkg.SourceGetCurrent(false).map do |repo|
+          Yast::Pkg.SourceGeneralData(repo)
+        end
+
+        pp repos
```

Comment:
this looks like debug relict

Reply:
no, this command really prints the data, it's for the command line console